### ISSUE TITLE
(Self-Voicing) Full details of a discharge event

### ISSUE THREAD

**Issue by terracoda** [phetsims/john-travoltage#354]
Following up from my https://github.com/phetsims/john-travoltage/issues/353#issuecomment-704208126, @Matthew-Moore240 and I need to review how the details of a discharge event are communicated in the Interactive Description design.

I think the position of the hand is the detail that is missing.

Assigning us both, so who ever gets to this first.

Basically, I am wondering if the missing detail or details are delivered in a response or in the state description of the screen summary. This will help us determine where we deliver it for Self-Voicing.

**Comment by terracoda** [comment]
**Example Context Responses about charges & discharges:**
- Discharge occurred. Electrons on body decreased from 34 to 0 with hand at position -2, very close to doorknob.
- Electrons on body: 34
- Electrons on body: 16

**The dynamic part of the scene summary:**
- John has 0 charges on his body. John's hand is very close to doorknob, and he is ready to swing his leg to rub his foot on the rug.

Another possible scenario in interactive description is a partial discharge:
- Discharge occurred. Electrons on body decreased from 76 to 50 with hand at position -2, very close to doorknob.


**For Self-Voicing**
For Self-Voicing, the discharge response does not contain the position of the hand at the point at which the discharge occurs. Though, when the learner lets go of the hand, the latest position is read out. This can happen, at, during or after a discharge in Self-Voicing.

- {{Some}} electrons discharged.

**For Partial Discharges**
For partial discharges, the latest hand position can be spoken before the actual details of the discharge. 

This may make it hard to determine or verify a point of discharge. We should discuss how much this matters.

Possible new Context Response for partial or full discharge for Self-Voicing:
- {{Some}} electrons discharged with hand {{very close to}} door knob.

**Comment by terracoda** [comment]
@Matthew-Moore240, what do you think? Does that capture enough detail?

**Comment by Matthew-Moore240** [comment]
Yeah I think thats great!

**Comment by jessegreenberg** [comment]
Done in the above commit, @terracoda @Matthew-Moore240 can you please review?

**Comment by terracoda** [comment]
I like the added detail in the discharge response.
Is it all good @Matthew-Moore240?

**Comment by Matthew-Moore240** [comment]
All good! Closing
